commands/role-request.js

```diff
@@ -0,0 +1,39 @@
+const {
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonStyle,
+  SlashCommandBuilder,
+} = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
```

Comment:
Let's add permissions so that only administrators can run the command. You can do this via `.setDefaultMemberPermissions(PermissionsBitField.Administrator)` on your SlashCommandBuilder